src/flamenco/vm/syscall/fd_vm_syscall_pda.c

```diff
@@ -33,7 +33,7 @@ fd_vm_derive_pda( fd_vm_t *           vm,
                   uchar *             bump_seed,
                   fd_pubkey_t *       out ) {
 
-  fd_vm_vec_t const * seeds_haddr = FD_VM_MEM_SLICE_HADDR_LD( vm, seeds_vaddr, FD_VM_ALIGN_RUST_U8,
+  fd_vm_vec_t const * seeds_haddr = FD_VM_MEM_SLICE_HADDR_LD( vm, seeds_vaddr, FD_VM_ALIGN_RUST_SLICE_U8_REF,
```

Comment:
I'm not sure this is correct - the alignment of the slices is effectively the alignment of the _elements_.

Reply:
https://github.com/firedancer-io/agave/blob/b641c6e050d759e41dd7d794c3d0a12638c11800/programs/bpf_loader/src/syscalls/mod.rs#L721-L722

Agave uses a &[u8] for the translation type